src/applications/claims-status/package.json

```
@@ -1,5 +1,5 @@
 {
-    "name": "@department-of-veterans-affairs/claims-status",
+    "name": "applications-claims-status",
```

```suggestion
    "name": "@department-of-veterans-affairs/applications-claims-status",
```
For the sake of consistency, let's also keep the `@department-of-veterans-affairs`. In the event another app eventually does import hooks or utilities from claims-status, they'd do so with `@department-of-veterans-affairs/applications-claims-status` which is consistent with how we name other VA packages.

---

updated